Automattic/woocommerce-payments
# Bump WC minimum supported version for 4.6.0 release

#### Changes proposed in this Pull Request

- Bump minimum WC from 6.0.1 to 6.2.2
- Add [WC 6.7.0](https://github.com/woocommerce/woocommerce/releases/tag/6.7.0).

No WP updates to catch up on.

#### Testing instructions

- Verify that wherever we used 6.0 as a minimum before, we now use 6.2.
- Please check the CI tests in the GH workflows.

-------------------


**Post merge**
